packages/angular/src/tracing.ts

```diff
@@ -47,8 +56,35 @@ export function routingInstrumentation(
   }
 }
 
+/**
+ * Creates routing instrumentation for Angular Router.
+ *
+ * @deprecated Use `browserTracingIntegration()` instead, which includes Angular-specific instrumentation out of the box.
+ */
+// eslint-disable-next-line deprecation/deprecation
 export const instrumentAngularRouting = routingInstrumentation;
 
+/**
+ * A custom BrowserTracing integration for Angular.
+ *
+ * Use this integration in combination with `TraceService`
+ */
+export function browserTracingIntegration(
+  options: Parameters<typeof originalBrowserTracingIntegration>[0] = {},
+): Integration {
+  // If the user opts out to set this up, we just don't initialize this.
+  // That way, the TraceService will not actually do anything, functionally disabling this.
+  if (options.instrumentNavigation === false) {
```

Comment:
I figured this is good enough to allow users to disable this. It means that if users do this:

```js
browserTracingIntegration({ instrumentNavigation: false })
```

+ add the TraceService, they will get the warning "Angular integration has tracing enabled, but Tracing integration is not configured". Id' say that's good enough?